website/docs/externaldata.md

```diff
@@ -452,13 +452,15 @@ server := &http.Server{
 
 ### Authenticate the API server against Webhook (Self managed K8s cluster only)
 
+> ⚠️ Two new flags will be introduced in v3.11 because of these changes. And since they are not backward compatible, you may need a clean install to make use of them.
+
 **Note:** To enable authenticating the API server you have to be able to modify cluster resources. This may not be possible for managed K8s clusters.
 
 To ensure a request to the Gatekeeper webhook is coming from the API server, Gatekeeper needs to validate the client cert in the request. To enable authenticate API server, the following configuration can be made:
 
-1. Deploy Gatekeeper with a client CA cert name. Provide name of the client CA with the flag `--client-cert-name`. The same name will be used to read certificate from the webhook secret. The webhook will only authenticate API server requests if client CA name is provided with flag.
+1. Deploy Gatekeeper with a client CA cert name. Provide name of the client CA with the flag `--client-ca-name`. The same name will be used to read certificate from the webhook secret. The webhook will only authenticate API server requests if client CA name is provided with flag.
```

Comment:
Might be good to provide an example of how this is configured as part of the deployment yaml and/or helm chart. might be good to update https://github.com/open-policy-agent/gatekeeper-external-data-provider to reflect this once v3.11 is out.